projects/wolfssl/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# variables
+target_dir="$SRC/fuzz-targets"
+LDFLAGS=""
+LDLIBS="-lwolfssl -lFuzzingEngine"
+
+
+
+# build project
+./autogen.sh
+./configure --enable-static --disable-shared --prefix=/usr CC="clang"
+make -j$(nproc) all
+make install
+
+
+
+# set up fuzz targets
+for each in "${target_dir}"/*/target.c; do
```

Comment:
We recommend that this be part of the upstream Makefile to make integration with development workflows easier and to prevent bitrot.